components/pages/searchPageLayout/SearchPageLayout.tsx

```diff
@@ -45,7 +49,11 @@ const branchIds = [
   "775130",
 ] as string[]
 
-export const formatFacetTerms = (filters: { [key: string]: { [key: string]: FacetValue } }) => {
+const SEARCH_RESULTS_LIMIT = 9
```

Comment:
This is an example of how it can go wrong when we do not have a fixed places for configuration. Here it is 9 but the limit defined in `app/search/fetchSearchResult.server.ts` is 10.

Reply:
...but the configuration part will be solved in the near future.